### PR TITLE
COMMERCE-12122 UOM ModelListener for Order

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/COMMERCE-12122

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/model/listener/CPInstanceUnitOfMeasureModelListener.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+			CommerceOrderConstants.ORDER_STATUSES_PROCESSING,
+			CommerceOrderConstants.ORDER_STATUSES_SHIPPING);
+
+		for (CommerceOrderItem commerceOrderItem :
+				_commerceOrderItemLocalService.
+					getCommerceOrderItemsForCommerceOrdersWithStatuses(
+						cpInstanceId, orderStatuses, originalKey,
+						QueryUtil.ALL_POS, QueryUtil.ALL_POS)) {
+
+			commerceOrderItem.setUnitOfMeasureKey(key);
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/model/listener/CPInstanceUnitOfMeasureModelListener.java:88]: what about the quantity?

**Review by andrea-ale-sbarra — APPROVED** [review]
overall looks good, just let's understand what to do with the quantity.

Thank you!

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
please correct the test scenario messages

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/model/listener/CPInstanceUnitOfMeasureModelListener.java (hunk 1 above)
Comment (smottal) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/model/listener/CPInstanceUnitOfMeasureModelListener.java:111]: Now we multiply the quantity with the incrementalOrderQuantity

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

**Comment by smottal** [comment]
ci:test:sf

**Pull request closed without merging**